### DISCUSSION TITLE
CA-BC & CA-AB System Operator Data Source

### DISCUSSION THREAD

**Discussion by D-Jeffrey** [electricitymaps/electricitymaps-contrib#4718] [Ideas]
Here are a few data sources to look at for BC:
https://www.bchydro.com/info/nsi-data/BCHNSI.xml
https://www.bchydro.com/energy-in-bc/operations/transmission/transmission-system/actual-flow-data.html

For AESO, they have an API
https://api.aeso.ca/web/api/ets and a way to see the energy sources broken down by sources and facility names http://[ets.aeso.ca/ets_web/ip/Market/Reports/CSDReportServlet](http://ets.aeso.ca/ets_web/ip/Market/Reports/CSDReportServlet)

Some of those system facilities can be found here: https://transalta.com/about-us/our-operations/facilities/ others https://www.capitalpower.com/operations/

**Comment by madsnedergaard** [comment]
Thanks @D-Jeffrey, we have this issue for CA-BC: https://github.com/electricitymaps/electricitymaps-contrib/issues/3801
If there's real-time production data available for the zone now, please write in the issue with the relevant links - otherwise I'm afraid there's no much more we can do right now :)
